contracts/GSN/IRelayRecipient.sol

```diff
@@ -24,7 +42,32 @@ contract IRelayRecipient {
         view
         returns (uint256, bytes memory);
 
+    /**
+     * @dev Called by {IRelayHub} on approved relay call requests, before the relayed call is executed. This allows to e.g.
```

Comment:
Is the sentence meant to have e.g. at the start rather than part way through?
```suggestion
     * @dev Called by {IRelayHub} on approved relay call requests, before the relayed call is executed. e.g. This allows to
```